<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1721</width>
    <height>1162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>821</width>
      <height>411</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Image 1</string>
     </property>
    </widget>
    <widget class="ImageView" name="Image1_Area" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="ImageView" name="Image_1_Components_Area" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="Image_1_Components">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>40</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Imaginary</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Open_Image_1">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>93</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>821</width>
      <height>411</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Image 2</string>
     </property>
    </widget>
    <widget class="ImageView" name="Image2_Area" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="ImageView" name="Image_2_Components_Area" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="Image_2_Components">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>40</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Imaginary</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Open_Image_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>93</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>821</width>
      <height>411</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mixer Output to:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Output 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Output 2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Component 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Component 2</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Image 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Image 2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>120</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Imaginary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uniform Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uniform Phase</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Image 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Image 2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_7">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>250</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Imaginary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uniform Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uniform Phase</string>
      </property>
     </item>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>591</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>591</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>450</y>
      <width>821</width>
      <height>411</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="ImageView" name="Output1_Area" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="ImageView" name="Output2_Area" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Output1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Output 2</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
